libavogadro/src/extensions/molecularpropextension.cpp

```diff
@@ -207,8 +210,23 @@ namespace Avogadro {
       m_molecule->setProperty("name", QVariant()); // set an invalid name, since we don't have one
   }
 
+  void MolecularPropertiesExtension::printSslErrors(
+                                                QNetworkReply*, 
+                                                const QList<QSslError> &errors)
+  {
+    foreach(const QSslError &error, errors) {
+      qDebug() << QString("SSL Error: %1").arg(error.errorString());
```

Comment:
Even though this is debugging, I think we'd want to use tr("SSL Error %1") to allow localization.